### DISCUSSION TITLE
Compilation warning

### DISCUSSION THREAD

**Discussion by AgentCousto** [aws/aws-sdk-cpp#2835] [Q&A] [answered]
Hello,

Since I updated my compiler, I have the following compilation warnings:
![image002 (1)](https://github.com/aws/aws-sdk-cpp/assets/151918897/6bcce840-990e-4d02-b229-dd6d223f8129)

Do you know if this is coming from the AWS SDK code? If yes, will it be updated?

Thank you for your help.

**Comment by jmklix** [comment]
What compiler and version are you using? and what version did you update from?

Also what OS are you using?

**Reply by AgentCousto** [reply]
Hello,

I am using the MSVC v143 compiler and the aws sdk version 1.9.365

**Reply by AgentCousto** [reply]
I am using Windows 10

**Reply by jmklix** [reply, marked as answer]
Which compiler version did you update from?

You can turn off warnings as errors by setting this cmake argument to off:  `AWS_SDK_WARNINGS_ARE_ERRORS`. That might fix your problem.

I would also recommend updating from version `1.9` to the latest version of this sdk.
